### DISCUSSION TITLE
[Question] Is there a list of all Ubuntu dependencies needed for build?

### DISCUSSION THREAD

**Discussion by iamthesenate1** [lxqt/lxqt#2011] [Q&A] [answered]
I ask this because when trying to build on an Ubuntu machine I kept getting errors like these 

>   Could not find a package configuration file provided by "KF5WindowSystem"
>   (requested version 5.16.0) with any of the following names:
> 
>     KF5WindowSystemConfig.cmake
>     kf5windowsystem-config.cmake

**Comment by tsujan** [comment, marked as answer]
→ https://github.com/lxqt/lxqt/wiki/Building-from-source

**Reply by iamthesenate1** [reply]
Thanks very much! I now feel stupid for not looking there.
